crates/view/src/worker.rs

```diff
@@ -303,21 +303,72 @@ impl Worker {
                                 // Update the position record
                                 self.storage.update_position(position_id, state).await?;
                             }
+                            penumbra_transaction::Action::ActionDutchAuctionSchedule(
```

Comment:
why do we need these action checks?

Reply:
On advice, we scan transactions and their actions, and record auction metadata by their asset id. This is useful to be able to display relevant asset metadata when presented with an asset id (among other things).

Reply:
For a little bit more context: `Value`s in the shielded pool are an `Amount` and an `AssetId`. The asset ID (an $\mathbb F_q$ value) is the currently always a hash of a denom string, and client tooling looks up `asset.Metadata` via the asset ID in an asset registry.

The chain maintains one minimal asset registry (just containing denoms), but this registry does _not_ include entries for all of the LPNFTs (and now ANFTs), because this would bloat the chain state. Instead, the client's view service is responsible for locally deriving and caching the metadata of the bearer tokens it creates. This saves data for all the other clients, who don't want to have to maintain knowledge of everyone else's position NFTs.